doc/COOKBOOK-FIXERS.md

```diff
@@ -128,7 +128,7 @@ final class RemoveCommentsFixerTest extends AbstractFixerTestCase
 
     public function provideFixCases()
     {
-        return array();
+        return [];
```

Comment:
that's why atomic commits are nice, it would be only one click if you would commit your work atomically ;)